### DISCUSSION TITLE
MMS icon in O2 Brand

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1830] [Proposals]
### Goal

Get an icon to represent MMS messages.

### Context

We need to differentiate between SMS and MMS message.

### Impact

Feed the O2 icon library.

### Discussion Questions

Could we have this new icon?

### Do you have a visual proposal?

https://www.figma.com/design/qKsUe3xW8wDOoySLOzWw3I/My-O2-App-Master-2024?node-id=1680-112447&t=SBFI7BIGirh3f1Zh-4

**Comment by yceballost** [comment]
nice! I will create a issue and include it in the next release

thx <3
